cdap-ui/app/directives/instance-control/instance-control.js

```diff
@@ -21,18 +21,28 @@ angular.module(PKG.name + '.commons')
       restrict: 'E',
       controller: 'instanceControlController',
       scope: {
-        basePath: '='
+        basePath: '=',
+        isWorker: '@'
```

Comment:
I think this should be more generic. 
- Either we should have a separate directive that uses the instance-control generic directive and adds any worker specific logic there
- Or we should make this generic if backend is not fixing the response. Right now if I am using this instance control in flow and if I want don't have requested from backend then having `is-worker` set to true doesn't give a better picture of what that attribute is doing.